dev/com.ibm.ws.transport.http/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -303,8 +303,9 @@
         <AD name="%http.samesite.strict" description="%http.samesite.strict.desc"
            id="strict" required="false" type="String" cardinality="5000"/>
 
-        <AD name="internal" description="internal use only"
-           id="partitioned" required="false" type="Boolean" default="false"/>
+        <AD name="%http.samesite.partitioned" description="%http.samesite.partitioned.desc"
+           id="partitioned" required="false" type="Boolean" default="false" ibm:beta="true">
```

Comment:
This looks to be causing the failures in your build, you need to end `<AD` with `/>`